src/components/AppBar.tsx

```diff
@@ -25,15 +25,15 @@ const AppBar: React.FC = () => {
         >
           Fresh Market
         </Text>
-        {route.name != RoutesName.products ||
-          (route.name != RoutesName.processing && (
+        {route.name != RoutesName.products &&
+          route.name != RoutesName.processing && (
```

Comment:
When comparing wouldn't be better to compare value and type(!==) instead of just value(!=)?

Reply:
Yeah, it's ideal to have === compares to avoid bugs, most of the time you don't need to compare only values.

Reply:
I'm still a little bit used with flutter 😅